src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -251,11 +240,84 @@ public Event messageID(MessageID attachedMessageID) {
         return this;
     }
 
-    public List<com.exactpro.th2.common.grpc.Event> toListProto(@Nullable EventID parentID) throws JsonProcessingException {
-        return collectSubEvents(new ArrayList<>(), parentID);
+    public List<com.exactpro.th2.common.grpc.Event> toListProto(@NotNull EventID parentId) throws JsonProcessingException {
```

Comment:
Please apply the requiredNotNull method for all NotNull variable

Reply:
Also, string values should not be blunk

Reply:
done